lms/djangoapps/courseware/features/problems.feature

```diff
@@ -125,25 +125,25 @@ Feature: LMS.Answer problems
         Then I should see a score of "<Points Possible>"
 
         Examples:
-        | ProblemType       | Correctness   | Score               | Points Possible    |
-        | drop down         | correct       | 1/1 point           | 1 point possible   |
-        | drop down         | incorrect     | 1 point possible    | 1 point possible   |
-        | multiple choice   | correct       | 1/1 point           | 1 point possible   |
-        | multiple choice   | incorrect     | 1 point possible    | 1 point possible   |
-        | checkbox          | correct       | 1/1 point           | 1 point possible   |
-        | checkbox          | incorrect     | 1 point possible    | 1 point possible   |
-        | radio             | correct       | 1/1 point           | 1 point possible   |
-        | radio             | incorrect     | 1 point possible    | 1 point possible   |
-        #| string            | correct       | 1/1 point           | 1 point possible   |
-        #| string            | incorrect     | 1 point possible    | 1 point possible   |
-        | numerical         | correct       | 1/1 point           | 1 point possible   |
-        | numerical         | incorrect     | 1 point possible    | 1 point possible   |
-        | formula           | correct       | 1/1 point           | 1 point possible   |
-        | formula           | incorrect     | 1 point possible    | 1 point possible   |
-        | script            | correct       | 2/2 points          | 2 points possible  |
-        | script            | incorrect     | 2 points possible   | 2 points possible  |
-        | image             | correct       | 1/1 point           | 1 point possible   |
-        | image             | incorrect     | 1 point possible    | 1 point possible   |
+        | ProblemType       | Correctness   | Score                        | Points Possible               |
+        | drop down         | correct       | 1/1 point earned (ungraded)  | 1 point possible (ungraded)   |
+        | drop down         | incorrect     | 1 point possible (ungraded)  | 1 point possible (ungraded)   |
+        | multiple choice   | correct       | 1/1 point earned (ungraded)  | 1 point possible (ungraded)   |
+        | multiple choice   | incorrect     | 1 point possible (ungraded)  | 1 point possible (ungraded)   |
+        | checkbox          | correct       | 1/1 point earned (ungraded)  | 1 point possible (ungraded)   |
+        | checkbox          | incorrect     | 1 point possible (ungraded)  | 1 point possible (ungraded)   |
+        | radio             | correct       | 1/1 point earned (ungraded)  | 1 point possible (ungraded)   |
+        | radio             | incorrect     | 1 point possible (ungraded)  | 1 point possible (ungraded)   |
+        #| string            | correct       | 1/1 point earned (ungraded) | 1 point possible (ungraded)   |
```

Comment:
If it is not required and is not a commentary on the code we should be able to remove it.  Worse case someone can pull it up in the history if they needed it.